### DISCUSSION TITLE
Create metrically equivalent font?

### DISCUSSION THREAD

**Discussion by fonttools-user** [fonttools/fonttools#3079] [Q&A]
I have a font A and a font B
I want to create a font C with the glyphs from font A but with the metrics from font B. Additionally, I want all characters not present in font B to be removed from font C and I want to be warned of characters present in font B but not in font A.

**Comment by isus-iwiwisovsky** [comment]
Hello! I also have this issue

**Reply by isus-iwiwisovsky** [reply]
@anthrotype can you help

**Comment by ghuser1234567890** [comment]
When will this be answered?

**Reply by ghuser1234567890** [reply]
@justvanrossum @anthrotype

**Reply by ghuser1234567890** [reply]
@behdad @brawer

**Reply by ghuser1234567890** [reply]
Can any of you help?

**Reply by justvanrossum** [reply]
Will you keep on making new one-off github accounts to ask people to do your homework?
